lib/tlog/tty_sink.c

```diff
@@ -51,6 +51,20 @@ tlog_tty_sink_is_valid(const struct tlog_sink *sink)
     return tty_sink != NULL;
 }
 
+static void
+tlog_tty_sink_io_close(struct tlog_sink *sink, bool output)
+{
+    struct tlog_tty_sink *tty_sink =
+                                (struct tlog_tty_sink *)sink;
+
+    int *fd = output ? &tty_sink->out_fd : & tty_sink->in_fd;
```

Comment:
There's an extra space after `&` here, which makes it look like the binary AND operation.